cpp/ps0000/main (1).cpp

```diff
@@ -1,11 +1,12 @@
-#inline <iostream>
+#include <iostream> // 2/2
 using namespace std;
-int main()
+int main() // 1/1
 {
-    :cout << "Hallo world";
-//@ cannot be used instead of # 
-// cannot be used int main(void)
-//cannot be used iostream.h
-//Not 1 but 0 
-return 0
-}
\ No newline at end of file
+    cout << "Hello, World\n"; // 2/4
```

Comment:
- [x] used `<<` for outputting instead of `>>`
- [ ] used `\n` or `std::endl` in the right way
- [x] avoided syntax error by typing `;` at the end of the line
- [ ] output is as required `Hello, World!` in a single line